### DISCUSSION TITLE
Mineria Dual ETH + ZIL error

### DISCUSSION THREAD

**Discussion by Tonitkd** [Lolliedieb/lolMiner-releases#854] [Q&A]
Buenas gente como estan? estoy queriendo hacer mineria dual, venia perfecto con otra pool, en Flexpool, tenia estas configuraciones y hacia mineria dual con ETH en esa Pool y ZIL en EZIL

![image](https://user-images.githubusercontent.com/89988162/141181373-42da33ae-7e00-45e3-8a11-8153cc4a012a.png)

Pero ahora quiero cambiar de pool, por Crazypool y me tira estos errores la configuracion

![image](https://user-images.githubusercontent.com/89988162/141181455-0b57ee5e-f6f3-4bab-b835-a552f4bce112.png)
![image](https://user-images.githubusercontent.com/89988162/141181491-fa90b1ad-45fb-4ce4-9a18-6c92d71957c0.png)

Me podran ayudar? muchas gracias!

**Comment by WoozieJuank** [comment]
Hola! lo pudiste resolver?
